frontend/src/app/admin/page.tsx

```diff
@@ -0,0 +1,21 @@
+"use client";
+import Dashboard from "@/components/dashboard/dashboard";
+import Navbar from "@/components/navbar/navbar";
+import { useEffect, useState } from "react";
+
+export default function Home() {
+  const [name, setName] = useState<string>("");
+  useEffect(() => {
+    const storedName = localStorage.getItem("name")
+      ? String(localStorage.getItem("name"))
+      : "";
+    setName(storedName);
```

Comment:
```suggestion
    if (localStorage.getItem("user") {
        setName(JSON.parse(localStorage.getItem("user")).name)
    }
```

This might not work off the bat but something like this